protractorFailuresReport/protractorTestErrors-1.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This shouldn't be in the commit. I'll add it to the git ignore.

Reply:
Whoops! Removed it. Can we send those into the /test/ directory too?